tests/test_header.py

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+"""
+Test suite for the docx.header module
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import pytest
+
+from docx.header import Header
+
+from .unitutil.cxml import element
+
+
+class DescribeSection(object):
+    def it_knows_its_linked_previous(self, header_linked_previous_fixture):
+        header, expected_linked_previous = header_linked_previous_fixture
+        assert header.is_linked_to_previous is expected_linked_previous
+
+    # fixtures -------------------------------------------------------
+
+    @pytest.fixture(params=[
+        ('w:sectPr',                                    True),
+        ('w:sectPr/w:headerReference{w:type=default}',  False),
+        ('w:sectPr/w:headerReference{w:type=odd}',      True),
+    ])
+    def header_linked_previous_fixture(self, request):
+        sectPr_cxml, expected_linked_previous = request.param
+        sectPr = element(sectPr_cxml)
+        header = Header(sectPr)
+        return header, expected_linked_previous
```

Comment:
* On the future includes, I generally include division as well on new modules (and when I think of it when in existing modules). I've found it too hit-or-miss to remember on first use.
* This is a test on Header, so it's DescribeHeader, not DescribeSection
* blank space between class decl and first method decl. No docstrings on these so it's a bit different from convention on library code.
* The general convention on fixtures is x_fixture, where x is exactly the property or method name. So this would be is_linked_to_previous_fixture. However, space is at a premium, so if a shorter version works for a long one like this it's generally a good idea. I'd say 'is_linked_fixture'.
* expected_linked_previous should be expected_value. That varies throughout the tests, I originally tried to match, but ended up standardizing on just expected_value.
* since sectPr is not used again in the fixture, I generally put the 'element(sectPr_cxml)' inline into the Header() call.